pom.xml

```diff
@@ -36,13 +40,13 @@
   <repositories>
     <repository>
       <id>repo.jenkins-ci.org</id>
-      <url>http://repo.jenkins-ci.org/public/</url>
+      <url>https://repo.jenkins-ci.org/public/</url>
```

Comment:
Not true. You need to define at least one of them (`repostories` IIRC) just to make sure you can *find* the parent POM, on an otherwise unconfigured system.